### DISCUSSION TITLE
Export response rules etc. to Mockoon CLI?

### DISCUSSION THREAD

**Discussion by royderegt** [mockoon/mockoon#684] [Help] [answered]
Hi all,

I've been using Mockoon in a docker environment for the last two weeks, and it's been working great. I use the Mockoon GUI to export my settings and I start the docker with this file passed as argument to the `--data` flag. Right now I'm trying to simulate an unstable server, using multiple responses for a single endpoint with either rules to determine which response to give or using 'sequential response'. However, I can't seem to export these settings into an OpenAPI spec. 
How can I simulate an unstable server via Mockoon CLI?

Thanks!

**Comment by 255kb** [comment, marked as answer]
Hi Roy,

You will find in the [CLI documentation](https://github.com/mockoon/mockoon/tree/main/packages/cli) that you can directly use Mockoon's environment files and that the CLI supports all features that you will find in the UI.
However, by exporting in the Open API format you will lose some information as Open API does not have the same features than we have in Mockoon, and vice versa. This is true for the rules and other Mockoon specific features.
